db/schema.rb

```diff
@@ -10,19 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_03_31_095756) do
+ActiveRecord::Schema.define(version: 2022_04_05_121718) do
 
-  create_table "coronavirus_announcements", charset: "utf8", force: :cascade do |t|
-    t.bigint "coronavirus_page_id", null: false
-    t.string "title", null: false
-    t.text "url", null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.integer "position", null: false
-    t.date "published_on"
-  end
-
-  create_table "coronavirus_pages", charset: "utf8", force: :cascade do |t|
+  create_table "coronavirus_pages", charset: "utf8mb3", force: :cascade do |t|
```

Comment:
I wondered why the charset changed, but it seems like utf8 is an alias of utf8mb3 so should be fine.

Reply:
Yeah, this was automatic when I did `rails db:migrate` locally - news to be but seems to be a [MySQL specific](https://dev.mysql.com/doc/refman/8.0/en/charset-unicode-utf8mb3.html) charset.